walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,22 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
```

Comment:
These methods can both be static and then you could disallow the constructor of this class